include/libtorrent/peer_connection_handle.hpp

```diff
@@ -57,7 +58,7 @@ struct TORRENT_EXPORT peer_connection_handle
 		: m_connection(impl)
 	{}
 
-	int type() const;
+	connection_type type() const;
```

Comment:
actually, technically it is public. it's not in the ``aux`` namespace nor in the ``aux_`` directory. Once ``peer_connection.hpp`` gets moved, this type should remain though